src/wp-includes/class-wp-image-editor-imagick.php

```diff
@@ -444,6 +444,39 @@ protected function thumbnail_image( $dst_w, $dst_h, $filter_name = 'FILTER_TRIAN
 		}
 
 		try {
+			$is_png                                      = false;
```

Comment:
do we really need all these extra variables? as far as I can tell these are only used once, which makes adding them excessive. can we just remove this part entirely, it seems unrelated.